docs/Tutorials/Advanced/ticketing/drop.md

```diff
@@ -0,0 +1,390 @@
+
+This is done with `NEAR-API-JS` and consists of:
+
+1) Create a Keystore, which stores your access keys used to sign transactions   
+  * select a network, either `testnet` or `mainnet`  
+  * choose a location where the keypairs live, either a folder on your local machine, or in-memory      
+
+2) Define a NEAR configuration using the Keystore  
+3) Use the configuration to initialize a connection to NEAR  
+
+More information about this process can be found [here](https://docs.near.org/tools/near-api-js/quick-reference#key-store).
+
+:::note
+For simplicity, this tutorial will choose a file-based keystore and point to the `~/.near-credentials` folder on your local machine since this is where most of your keys are stored. For more information about KeyStores, visit NEAR's [official docs](https://docs.near.org/tools/near-api-js/quick-reference#key-store).
+:::
+
+```js reference
+https://github.com/keypom/keypom-js/blob/5e4b4744a16c727d96d235282020c186edd0b0b5/docs-advanced-tutorials/ticket-app/frontend/utils/createTickDrop.js#L9-L26
+```
+
+---
+
+## Creating Drop with Function Call Data
+In this section, you'll create the function call drop to meet the functional requirements defined earlier.
+
+Recall that the drop needs the following properties:
+
+* The drop must have 2 key uses for each key.
+* The second key use should call `nft_mint`.
+* the first key use must be password protected.
+
+### Function Call Drop Basics
+This process starts with calling the `initKeypom` function and will always be the first function you call to interact with the SDK. 
+
+`initKeypom` initializes the SDK to allow for interactions with the Keypom smart contracts. Without it, none of the other SDK functions would work as expected. If a NEAR connection is not already present, it will initialize a new one for you. More info on the `initKeypom` function can be found [here](../../../keypom-sdk/modules#initkeypom).
+
+After `initKeypom` is called, the FC Drop can be created by calling `createDrop` and adding an `fcData` parameter. 
+
+:::tip
+Recall that the private keys being generated using `createDrop` are used to store the assets. These keys are then embedded within a link.
+:::
+
+The primary task in creating the Function Call Drop is to define `fcData`. It is an object with the following properties.
+
+```bash
+fcData
+├── methods
+```
+
+- `FCConfig`: [Specific configurations](../../../keypom-sdk/interfaces/FCConfig.md) for the FC Drop.  
+- `methods`: A vector of all the functions to be called for each key use.  
+
+In this example, only the `methods` parameter will be defined for the sake of simplicity.
+
+`methods` is a 2D array. The outer array defines the vectors of functions to be called **per key use**. The inner array dictates the functions invoked on **each particular key use**. For more information on the `methods` parameter, please see the [TypeDocs](../../../keypom-sdk/interfaces/Method.md)
```

Comment:
```suggestion
For multi-use keys, each specific use can have a different set of methods that will be called. These methods are executed sequentially and not in parallel. As an example, a key with 3 uses can be seen:
1. `nft_mint`
2. `null`
3. `create_account_advanced`, `setup`, `nft_mint` 
The first time the key is used, an NFT will be minted. The second use will simply advance the key and no method will be called. The third time the key is used, it will first call `create_account_advanced`. Once that's finished it will call the `setup` method and then finally `nft_mint`.

For more information on the `methods` parameter, please see the [TypeDocs](../../../keypom-sdk/interfaces/Method.md)
```